scss/boost_union/post.scss

```diff
@@ -1185,68 +1190,195 @@ body.theme-boost-union-commincourse {
             width: calc(98% - #{$blockregionoutsiderightwidth} - #{$blockregionoutsideleftwidth});
         }
 
-        /* Set the width and margin of the outside-left region. */
-        #theme-block-region-outside-left {
-            margin-right: 1rem;
-            margin-top: 1.5rem;
-            width: $blockregionoutsideleftwidth;
+        /* Set the width of the outside-* regions to auto.
+           Using a 100% width in the main inner area would push the side regions outside the page. */
+        #page .main-inner-outside-left-right .main-inner,
+        #page .main-inner-outside-left .main-inner,
+        #page .main-inner-outside-right .main-inner {
+            width: auto;
```

Comment:
I think we can simplify that by moving that single rule into the

```
.main-inner-outside-left-right,
        .main-inner-outside-left,
        .main-inner-outside-right
```

bracked above.